panic: runtime error: index out of range

just tired this thing out, but got a panic immediately:

```
~/dev/src/github.com/segmentio/robofiles on master ∴ robo XXXXXXXX rpc http://XXXXXXXX-service.segment.local/rpc
http://XXXXXXXX-service.segment.local/rpc> 
panic: runtime error: index out of range

goroutine 1 [running]:
panic(0x875640, 0xc82000e070)
	/usr/local/go/src/runtime/panic.go:464 +0x3e6
github.com/segmentio/rpc-cli/internal/rpc.(*Command).interactive(0xc820107eb8, 0x0, 0x0)
	/home/ubuntu/src/github.com/segmentio/rpc-cli/internal/rpc/rpc.go:116 +0x354
github.com/segmentio/rpc-cli/internal/rpc.(*Command).Run(0xc820107eb8, 0x0, 0x0)
	/home/ubuntu/src/github.com/segmentio/rpc-cli/internal/rpc/rpc.go:45 +0x98
main.main()
	/home/ubuntu/src/github.com/segmentio/rpc-cli/cmd/rpc/main.go:42 +0x4a1
```

donno what i did wrong O_O

--- a/internal/rpc/rpc.go
+++ b/internal/rpc/rpc.go
@@ -113,6 +113,10 @@ func (c *Command) interactive() error {
 			return err
 		}
 
+		if len(args) == 0 {
+			continue
+		}
+
 		method := args[0]
 		req := request(args[1:])
 		err = c.call(method, req)

